cfgov/unprocessed/js/modules/jquery/cf_notifier.js

```diff
@@ -51,9 +49,13 @@ var _notifier = {
       state:   settings.state,
       icon:    icon[settings.state]
     };
-    var template = handlebars.compile( settings.template );
+    var template = settings.template.replace( /\{\{\s?(.*?)\s?\}\}/g,
+      function( match, token ) {
+        return data[token];
```

Comment:
If this function were to be generalized for use elsewhere it'd be nice to have it return an empty string instead of `undefined` if the token isn't found (i.e. `return data[token] || ''`) buuuut in the current scenario it works A-OK.